pkgs/by-name/ga/gate/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "gate";
+  version = "0.36.7";
+
+  src = fetchFromGitHub {
+    owner = "minekube";
+    repo = "gate";
+    rev = "v${version}";
+    hash = "sha256-WHxpx20O/HuCWqbY4zTxcjyIhW3+FQtTz5sUGAda71g=";
+  };
+
+  vendorHash = "sha256-dswNJQWqN+u/mnpbj9se2j9uEi0ewNTXVlN3WnNbcyg=";
+
+  ldflags = [ "-s" "-w" ];
```

Comment:
Go programs sometimes need an internal `version` set so that commands that display it work correctly. For example `"-X main.version=v${version}"` may be needed for a package. 

I haven't looked into gate yet, but could you check if there is anything similar that we can set?

Reply:
There doesn't seem to be a version subcommand/flag whatsoever in gate.

Reply:
Okay, just wanted to check.